MiniDSP.h

```diff
@@ -0,0 +1,184 @@
+/* Copyright (C) 2014 Kristian Lauszus, TKJ Electronics. All rights reserved.
+ This software may be distributed and modified under the terms of the GNU
+ General Public License version 2 (GPL2) as published by the Free Software
+ Foundation and appearing in the file GPL2.TXT included in the packaging of
+ this file. Please note that GPL2 Section 2[b] requires that all works based
+ on this software must also be made publicly available under the terms of
+ the GPL2 ("Copyleft").
+ Contact information
+ -------------------
+ Kristian Lauszus, TKJ Electronics
+ Web      :  http://www.tkjelectronics.com
+ e-mail   :  kristianl@tkjelectronics.com
+ */
```

Comment:
You can also update the copyright header to add yourself  :)


Also can you use this one instead: https://github.com/felis/USB_Host_Shield_2.0/blob/master/XBOXONESBT.h#L1-L16, as I do not use that email anymore.